tests/unit/utils/resolve-value-test.js

```diff
@@ -0,0 +1,56 @@
+import resolveValue from 'sl-ember-behavior/utils/resolve-value';
+import { module, test } from 'qunit';
+
+module( 'Unit | Utility | resolve value' );
+
+test( 'Returns the passed value if the value is not a function', function( assert ) {
```

Comment:
Need to test all possible types:

* Number
* String
* Array
* Object
* undefined
* Boolean
* null
* Symbol